src/screens/SecretIconsScreen.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { FlatList, StyleSheet, View } from 'react-native';
+
+import { RegularText } from '../components';
+import { colors, Icon, normalize } from '../config';
+
+type TIconItem = {
+  iconName: string;
+};
+
+const iconNames = Object.keys(Icon);
+
+const IconItem = ({ iconName }: TIconItem) => {
+  const SelectedIcon = Icon[iconName];
+
+  return (
+    <View style={styles.iconField}>
+      <SelectedIcon size={normalize(35)} color={colors.secondary} />
+      <RegularText smallest>{iconName}</RegularText>
+    </View>
+  );
+};
+
+export const SecretIconsScreen = () => {
+  return (
+    <FlatList
+      data={iconNames}
+      renderItem={({ item }) => <IconItem iconName={item} />}
+      keyExtractor={(item) => item}
+      numColumns={3}
+    />
+  );
+};
+
+const styles = StyleSheet.create({
+  iconField: {
```

Comment:
i am wondering about `field` here. i think we often use `wrapper` or `container`. could you think of using `iconWrapper` or `iconContainer` here for consistency?

Reply:
you're right, changed it to iconContainer [29186de](https://github.com/smart-village-solutions/smart-village-app-app/pull/914/commits/29186de714d420a15bf9284a4e17cbc553609203)